packages/react/src/components/Tab/Tab.js

```diff
@@ -66,6 +70,18 @@ export default class Tab extends React.Component {
      */
     onKeyDown: PropTypes.func.isRequired,
 
+    /**
+     * A handler that is invoked when a user clicks on the control.
+     * Reserved for usage in Tabs
+     */
+    onTabClick: PropTypes.func,
+
+    /**
+     * A handler that is invoked on the key down event for the control.
+     * Reserved for usage in Tabs
+     */
+    onTabKeyDown: PropTypes.func,
+
```

Comment:
For this component, would we want `handleTabClick` to go to `onClick` versus `onTabClick`?